azure/config/kubernetes.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
Is this an ARM template for the K8s cluster or do you use `AKS Engine` to create an ARM template and then deploy that?